src/github.com/stellar/federation/record.go

```diff
@@ -1,8 +1,12 @@
 package federation
 
-type Record struct {
-	Username  string `db:"username" json:"stellar_address"`
-	AccountId string `db:"account_id" json:"account_id"`
-	MemoType  string `db:"memo_type" json:"memo_type,omitempty"`
```

Comment:
But you aren't fetching the stellarAddress on reverse federation. You are fetching just the name part. Seems to better to be clear what is going on.